apps/studio/components/ui/FilterPopover.tsx

```diff
@@ -27,6 +27,9 @@ export const FilterPopover = ({
 
   const formattedOptions = activeOptions.map((option) => {
     const base = options.find((x) => x[valueKey] === option)
+    if (!base || !base[labelKey]) {
```

Comment:
Is this even something you changed? Seems kinda unrelated and might just wrongfully show up here?

Reply:
No, that's not my changes.